hack/cherry-pick-pull.sh

```diff
@@ -0,0 +1 @@
+/home/I311840/go/src/github.com/gardener/gardener-extension-provider-aws/vendor/github.com/gardener/gardener/hack/cherry-pick-pull.sh
```

Comment:
Is this change required? :o Mistake?

Reply:
Yes, but it should be relative path. It is fixed now.